extra-network/clash/autobuild/overrides/usr/lib/systemd/system/clash@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=A rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols for %i.
+After=network.target
+
+[Service]
+Type=exec
```

Comment:
Is this supposed to be a daemon (that constantly runs)? You might be looking for `Type=simple` or `Type=forking` here. See systemd.service's [documentation](https://freedesktop.org/software/systemd/man/systemd.service.html).